lib/LMSManagers/LMSConfigManager.php

```diff
@@ -134,12 +138,12 @@ public function GetConfigDefaultType($section, $var)
         return $type;
     }
 
-    public function CheckOption($type, $value)
+    public function CheckOption($var, $value, $section, $type)
```

Comment:
Tylko, że do tej metody `CheckOption()` będę musiał dodać i tak `$type` bo admin może zmienić `$type` przy edycji opcji i wtedy olewany jest defaultowy typ. Jeśli nie zmienimy definicji `CheckOption()` to zawsze będzie brana pod uwagę wartość zwracana z funkcji `GetConfigDefaultType()`.